protocols/1596/pcr_prep.ot2.py

```diff
@@ -0,0 +1,156 @@
+from opentrons import labware, instruments, modules, robot
+
+metadata = {
+    'protocolName': 'PCR Preparation',
+    'author': 'Nick <protocols@opentrons.com>',
+    'source': 'Custom Protocol Request'
+}
+
+# create custom labware
+pcr_plate_name = 'MicroAmp-96-PCR'
+if pcr_plate_name not in labware.list():
+    labware.create(
+        pcr_plate_name,
+        grid=(12, 8),
+        spacing=(9.025, 9.025),
+        diameter=5.5,
+        depth=22.5,
+        volume=200
+    )
+
+deepwell_plate_name = 'KingFisher-96-deepwell'
+if deepwell_plate_name not in labware.list():
+    labware.create(
+        deepwell_plate_name,
+        grid=(12, 9),
+        spacing=(9, 9),
+        diameter=8,
+        depth=38,
+        volume=1000
+    )
+
+tips_name = 'TipOne-tiprack-200ul'
+if tips_name not in labware.list():
+    labware.create(
+        tips_name,
+        grid=(12, 8),
+        spacing=(9, 9),
+        diameter=3.5,
+        depth=60
+    )
+
+# load labware and modules
+tempdeck = modules.load('tempdeck', '1')
+temp_plate = labware.load(pcr_plate_name, '1', share=True)
+if not robot.is_simulating():
```

Comment:
You don't need to check this, because during simulation, modules are not actually loaded which means `.set_temperature()` and other module methods would not have any effects